src/cbc_sdk/platform/alerts.py

```diff
@@ -1648,3 +1663,17 @@ def _perform_query(self, from_row=1, max_rows=-1):
             if current >= self._total_results:
                 still_querying = False
                 break
+
+    def close(self, closure_reason=None, determination=None, note=None, ):
```

Comment:
I'm still confused is reason we are doing this to prevent the inherited methods from working as the other workflow ones are inherited too